service/src/main/java/uk/gov/hmcts/reform/fpl/utils/CaseDetailsHelper.java

```diff
@@ -2,23 +2,22 @@
 
 import com.google.common.base.Splitter;
 import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
 import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
 import uk.gov.hmcts.reform.fpl.enums.State;
 
 import static uk.gov.hmcts.reform.fpl.enums.State.GATEKEEPING;
 import static uk.gov.hmcts.reform.fpl.enums.State.OPEN;
 import static uk.gov.hmcts.reform.fpl.enums.State.RETURNED;
 
+@Component
```

Comment:
Any particular reason for making it a component rather than leaving it as a static helper. I guess it does make testing easier as you can then more easily mock.

Reply:
yes, static methods are mockable now, but it's rather clunky and need some powermock magic. I personally don't like static, and it should be abolished from the java language. there's no reason why an helper can't be a normal object. :D